src/components/ChartBar/ChartBar.js

```diff
@@ -29,10 +29,20 @@ function ChartBar(props) {
     }
   };
 
+  function millBill() {
+    var str = state.updatedValue.toLocaleString();
+    var n = str.length;
+    
+    if (n <= 9 && n < 11) {
+    return 'Million';
+  } else { 
+    return 'Billion';
+  }
```

Comment:
This isn't working as expected because it's counting the commas (added by toLocaleString) in the string length. Change it to:
`var str = state.updatedValue.toString();`
and you get the correct number of digits!